src/System.Drawing.Graphics/src/System/Drawing/Graphics/Image.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+
+namespace System.Drawing.Graphics
+{
+    public enum PixelFormat
+    {
+        ARGB,
+        RGBA,
+        CMYK
+    }
+
+    public class Image
+    {
+            /* Private Fields */
+        private byte[] _data;
+        private int _width;
+        private int _height;
+
+            /* Properties */
+        public int WidthInPixels
+        {
+            get { return _width; }
+        }
+        public int HeightInPixels
+        {
+            get { return _height; }
+        }
+        public PixelFormat PixelFormat
+        {
+            get { return PixelFormat.RGBA; }
+        }
+        public byte[] Data
+        {
+            get { return _data; }
```

Comment:
> Returning the raw data can likely lead to a race condition as image manipulation is often not done on the UI thread while image rendering is. Is that a concern for this class?

I don't think so.  Here we are prototyping a library focused on server-side image manipulation.  See https://github.com/dotnet/corefx/issues/2020 for some of the context.

We do intend for the library to still work in client scenarios, and in that context you'd need to copy an image so you can render one copy and manipulate another.